src/test/java/org/folio/rest/impl/DataExportTest.java

```diff
@@ -86,6 +90,26 @@ public DataExportTest() {
     SpringContextUtil.autowireDependencies(this, vertxContext);
   }
 
+  @Test
+  void testExport_uploadingEmptyFile_FAILED_job(VertxTestContext context) throws IOException {
+    //given
+    String tenantId = okapiConnectionParams.getTenantId();
+    FileDefinition uploadedFileDefinition = uploadFile(EMPTY_FILE);
+    // when
+    ExportRequest exportRequest = buildExportRequest(uploadedFileDefinition);
+    postRequest(JsonObject.mapFrom(exportRequest), EXPORT_URL);
+    String jobExecutionId = uploadedFileDefinition.getJobExecutionId();
+    // then
+    vertx.setTimer(TIMER_DELAY, handler ->
+      jobExecutionDao.getById(jobExecutionId, tenantId).onSuccess(optionalJobExecution -> {
+        JobExecution jobExecution = optionalJobExecution.get();
+        context.verify(() -> {
+          assertJobExecution(jobExecution, FAIL, EXPORTED_RECORDS_EMPTY);
```

Comment:
Why is it asserted to null value? It must be 0, if you are creating a new Progress Object

Reply:
In my opinion, Progress should be created as a new object, and all fields should be 0

Reply:
Constant field value has been changed to 0 value

Reply:
@IlliaDaliek could you please test if csv file is empty